test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -186,6 +186,18 @@ Describe "Get-ChildItem" -Tags "CI" {
             $app.Target | Should -Not -Be $app.FullName
             $app.LinkType | Should -BeExactly 'AppExeCLink'
         }
+
+        It "Wildcard matching behavior is the same between -Path and -Include parameters when using escape characters" {
+            try {
+                New-Item -Type File 'a`[b]' -ErrorAction SilentlyContinue > $null
```

Comment:
Can we create the file under `$testdrive` no cleanup required then

Reply:
Done.